### ISSUE TITLE
B12.0 Refine AWS Registry Deployment Strategy

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/cloud-initiative#2]

**Comment by jordanpadams** [comment]
📆 April status: On schedule

**Comment by jordanpadams** [comment]
:calendar: May status: delayed due to issues with SA's deployment of Fargate in AWS. able to work other tasks in parallel as we await the deployment, so no impact on other tasks identified at this time.

**Comment by jordanpadams** [comment]
📆 June status: additional delays continue due to lack of SA resources to assist with Fargate / AWS issues. SA resource issues have been resolved, but 3 tasks are still delayed for preparing DN deployments. no foreseen impact on build deliverables at this time, but another delay may have impacts.

**Comment by jordanpadams** [comment]
📆 July status: delays per June setbacks. no foreseen impact on build deliverables at this time, but another delay may have impacts.

**Comment by jordanpadams** [comment]
All necessary B12.0 tasks completed. Other tasks can be deferred to B12.1.
